crates/relayer/src/channel.rs

```diff
@@ -857,7 +868,7 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> Channel<ChainA, ChainB> {
             State::Init,
             self.ordering,
             counterparty,
-            vec![self.dst_connection_id().clone()],
+            self.connection_hops.as_ref().unwrap().connection_ids(),
```

Comment:
The call to `.unwrap()` is failing in the test suite for non-multihop channels. I think it's because we need to use `veg![self.dst_connection_id()]` if `self.connection_hops` is `None`.

Reply:
```suggestion
            self.connection_hops.as_ref().map(|hops| hops.connection_ids()).unwrap_or_else(|| vec![self.dst_connection_id().clone()]),
```